src/components/Sidebar/index.js

```diff
@@ -42,26 +38,19 @@ const Sidebar = () => {
                 </Nav>
             </Navbar>
             <div className="social-icons">
-                <LogoGithub
-                    height="25px"
-                    width="25px"
-                />
-                <LogoLinkedin
-                    height="25px"
-                    width="25px"
-                />
-                <Mail
-                    height="25px"
-                    width="25px"
-                />
-                <LogoInstagram
-                    height="25px"
-                    width="25px"
-                />
-                <LogoBehance
-                    height="25px"
-                    width="25px"
-                />
+                <a href="https://github.com/ayu-ayushisood" target="_blank">
```

Comment:
Can we move these links a constants file ?

Reply:
yes. done